ls1intum/Artemis
# [Cypress] Setting up e2e testing with cypress

### Checklist

### Motivation and Context
 The increasingly great speed at which the artemis team is creating changes to the artemis platform causes an increased need for e2e testing. 

### Description
 I set up Cypress e2e testing using the briebug angular cypress schematic (https://www.npmjs.com/package/@briebug/cypress-schematic) and created some initial functionality and tests for it.

### Steps for Testing


1. Start local artemis server at :8080 or configure server target in cypress.json('baseUrl')
2. In artemis project in terminal enter:
2. yarn cypress open (to open the cypress testing browser)
3. yarn cypress run (to run the existing cypress tests in CLI)
4. should work without the yarn part the second time around
